libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/Conway/GovCert.hs

```diff
@@ -39,10 +39,11 @@ vStateSpec = constrained' $ \ [var|_dreps|] [var|_commstate|] [var|dormantepochs
 govCertSpec ::
   IsConwayUniv fn =>
   ConwayGovCertEnv (ConwayEra StandardCrypto) ->
-  VState (ConwayEra StandardCrypto) ->
+  CertState (ConwayEra StandardCrypto) ->
   Specification fn (ConwayGovCert StandardCrypto)
-govCertSpec ConwayGovCertEnv {..} vs =
-  let reps = lit $ Map.keysSet $ vsDReps vs
+govCertSpec ConwayGovCertEnv {..} certState =
+  let vs = certVState certState
```

Comment:
It doesn't seem to make sense to have a `CertState` argument if all we do is use the `VState`

Reply:
I'll leave it as it is for now, @lehins might had his reasons to do this despite what you point out is true.

Reply:
Yes, I do have my reasons :smile: 

We'll switch to the same approach for SPOs in the near future, so we'll need access to the `PState` as well.